spec/models/cloud_volume/cloud_volume_operations_spec.rb

```diff
@@ -0,0 +1,46 @@
+RSpec.describe 'CloudVolume::Operations' do
+  let(:ems)          { FactoryBot.create(:ems_vmware) }
+  let(:vm)           { FactoryBot.create(:vm_vmware, :ext_management_system => ems) }
```

Comment:
I don't think it makes sense to use a vmware ems or vm, particurally for a cloud_volume spec.  There are ems_cloud and vm_cloud factories that can keep things more generic.

Reply:
Done!